frontend/packages/console-shared/src/hooks/useCanClusterUpgrade.ts

```diff
@@ -0,0 +1,21 @@
+import { useAccessReviewAllowed } from '@console/dynamic-plugin-sdk';
+import { ClusterVersionModel } from '@console/internal/models';
+import { ClusterVersionKind, hasAvailableUpdates } from '@console/internal/module/k8s';
+
+export const isClusterExternallyManaged = () => {
```

Comment:
```suggestion
export const isClusterExternallyManaged = (): boolean => {
```

Reply:
Good catch, I added it last min and didn't properly vet my code. I'll update, thanks!